src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -646,7 +646,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!-- Determine the restore output path -->
     <PropertyGroup Condition=" '$(PackageReferenceCompatibleProjectStyle)' == 'true' OR '$(RestoreProjectStyle)' == 'ProjectJson' ">
-      <RestoreOutputPath Condition=" '$(RestoreOutputPath)' == '' " >$(MSBuildProjectExtensionsPath)</RestoreOutputPath>
+      <RestoreOutputPath>$(MSBuildProjectExtensionsPath)</RestoreOutputPath>
```

Comment:
I wouldn't change this. 
We recommend people not use it, but we just don't know who we'd end up breaking.

Is there a functional value to this or just a cleanup?

Reply:
Changing it didn't break any tests, so I guess it's only cleanup but will need to talk to sdk team about this.

I'll separate this into a new PR, we can make it in for .NET 5.